docs/docs/ref-10-glossary.md

```diff
@@ -189,5 +190,7 @@ type ReactComponent<TProps> = {
   props : TProps,
   render : () => ReactElement
 };
+
+type ReactStatelessComponent<TProps> = (TProps) => ReactElement;
```

Comment:
I wonder if we want to call this `ReactFunctionalStatelessComponent` or `ReactFunctionalComponent` instead. It’s stateless alright, but class components can also be stateless. Not sure.